src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -33,11 +33,12 @@ public class StaticWebAssetsIntegrationTest : MSBuildIntegrationTestBase, IClass
 
         public ITestOutputHelper Output { get; private set; }
 
-        [Fact(Skip = "https://github.com/dotnet/aspnetcore/issues/17233")]
+        [Fact]
```

Comment:
@NTaylorMullen, does this repo support quarantined tests?

Reply:
Not that I know of.